pkg/commands/engine/object.go

```diff
@@ -111,6 +111,40 @@ The JSON objects can be in either the GenericObjectProperties format or the Gene
 		},
 	}
 
+	var publishObjectCmd = &cobra.Command{
+		Use:     "publish <object-id>",
+		Args:    cobra.ExactArgs(1),
+		Short:   "Publish a generic object, like to make a sheet public",
+		Long:    "Publish a generic object, like to make a sheet public",
+		Example: "corectl object publish OBJECT-ID -a APP-ID",
+
+		Run: func(ccmd *cobra.Command, args []string) {
+			commandLineObjects := args[0]
+			if commandLineObjects == "" {
+				log.Fatalln("no objects specified")
+			}
+			ctx, _, doc, _ := boot.NewCommunicator(ccmd).OpenAppSocket(true)
+			internal.Publish(ctx, doc, args[0])
```

Comment:
Use the variable defined instead of `args[0]` here?

Reply:
does the variable need to be an array because it's args or are you saying just a string?

Reply:
No you already have a variable that is assigned as `commandLineObjects := args[0]`, so you can use that instead of `args[0]` since they are equivalent. And the variable is a string as `args` is a `[]string`.